packages/nuxt/src/common/types.ts

```diff
@@ -103,16 +103,16 @@ export type SentryNuxtModuleOptions = {
   debug?: boolean;
 
   /**
-   * Enabling basic server tracing can be used for environments where modifying the node option `--import` is not possible.
-   * However, enabling this option only supports limited tracing instrumentation. Only http traces will be collected (but no database-specific traces etc.).
+   * If this option is `false`, the Sentry SDK won't wrap the server entry file with `import()`. Not wrapping the
+   * server entry file will disable Sentry on the server-side. When you set this option to `true`, make sure
```

Comment:
```suggestion
   * server entry file will disable Sentry on the server-side. When you set this option to `false`, make sure
```

I think it should say `false` here?

Reply:
`true` is correct here, but I'll rephrase the JSDoc a bit so it is less confusing :)